casts/cards/src/UICollectionCard/types.ts

```diff
@@ -13,7 +13,7 @@ export type UICollectionCardProps = {
     /**
      * Displays the number of items in the collection or an image
      */
-    badge?: string | ImageSourcePropType;
+    badge?: string | number | ImageSourcePropType;
```

Comment:
Type `ImageSourcePropType` already contains type `number`. This is the type that the method `require` returns (at least on android). Consider to remove `number` here, so that there are no type collisions.

Reply:
Hm, but actually I think that it's OK to pass the number (which is not a part of ImageSource).
In general I don't think it's gonna have any collisions with types here..
@savelichalex what do you think?

Reply:
If you pass a picture to the component that you received via `require`, then this will break the component

Reply:
What's the problem with converting a number to a string to pass to badge?

Reply:
I don't think there're any problems, but I do agree with @SAnatoliiS that it's better to pass badge number as a string :)

Reply:
OK, hence this PR is not required ;) @vetkamak can you apply changes in Surf and close the PR?